templates/admission/doctorate/form_tab_project.html

```diff
@@ -110,15 +110,15 @@
     {% endpanel %}
 
     {% panel _("Doctoral project") %}
-        {% bootstrap_field form.titre_projet %}
-        {% bootstrap_field form.resume_projet %}
+        {% bootstrap_field form.titre_projet label=form.titre_projet.label|add:" *" %}
```

Comment:
```suggestion
        {% bootstrap_field form.titre_projet form_group_class="form-group required_field" %}
```
Ainsi que sur toutes les autres, ça te permet de dissocier la traduction du "métier"

Reply:
En effet ! Par contre je vois bien d'autres champs qui incluent cette classe `required_field` mais je n'ai pas de règles css appliquées :/

Reply:
j'ai ajouté une petite règle du coup, c'est beaucoup mieux. Reste à savoir si on met toutes ces petites règles dans un fichier CSS propre à admission ?